public/app/plugins/datasource/prometheus/datasource.test.ts

```diff
@@ -101,9 +101,10 @@ describe('PrometheusDatasource', () => {
 
   describe('Datasource metadata requests', () => {
     it('should perform a GET request with the default config', () => {
-      ds.metadataRequest('/foo');
+      ds.metadataRequest('/foo', { bar: 'baz' });
```

Comment:
Might be worth adding coverage for encoding and multiple parameters, e.g.:

```typescript
it('should perform a GET request with the default config', () => {
  ds.metadataRequest('/foo', { bar: 'baz baz', foo: 'foo' });
  expect(fetchMock.mock.calls.length).toBe(1);
  expect(fetchMock.mock.calls[0][0].method).toBe('GET');
  expect(fetchMock.mock.calls[0][0].url).toContain('bar=baz%20baz&foo=foo');
});
```

Reply:
I agree! I'll add multiple params.

Reply:
Added in https://github.com/grafana/grafana/pull/33321/commits/0f8f4a795fcebfda45bb4d19a47771454b6956a8